src/components/home/Reviews.tsx

```diff
@@ -1,11 +1,5 @@
-import Review from "./Review";
-
 const Reviews = () => {
-  return (
-    <div className="">
-      <Review />
```

Comment:
this should still be here

Reply:
the thing is it gives me an error, so i can't commit the changes. I have to leave them deleted